ocaml/xapi/xapi_stats.ml

```diff
@@ -44,7 +48,23 @@ let generate_master_stats ~__context =
         ~min:0.0 ~units:"sessions/s" ()
     )
   in
-  [session_count_ds; task_count_ds; session_count_change_ds]
+  let vgpu_migration_count =
+    Atomic.exchange pool_vgpu_migration_count 0 |> Int64.of_int
+  in
+  let vgpu_migration_count_ds =
+    ( Rrd.Host
+    , Ds.ds_make ~name:"pool_vgpu_migration_count"
+        ~description:"Number of vGPU migrations occurred since last update"
```

Comment:
Isn't this since last xapi start?

Reply:
All rate metrics are "since the last update". The wording confused me because I thought it was since the last time the host's packages were updated. I think the "since the last update" should be dropped.

Reply:
so note that "since last rrd update" is visible to users and they might not know what an rrd update is. In the ned this metric is a rate, and users don't care how it's calculated: the counter is reset every 5 seconds, or a difference is taken between the current measurement and the last one

Reply:
For comparison:
```
      , ds_make ~name:(key_format "iops_total")
          ~description:"I/O Requests per second"
          ~value:(Rrd.VT_Int64 (Int64.add value.iops_read value.iops_write))
          ~ty:Rrd.Absolute ~units:"requests/s" ~min:0. ()
```